BUILD.cmake.md

```diff
@@ -28,27 +28,46 @@ On macOS CMake may be installed by Homebrew/MacPorts or using the installer from
 
 To build HIDAPI as a standalone package, you follow [general steps](https://cmake.org/runningcmake/) of building any CMake project.
 
-An example of building CMake using Ninja:
+An example of building HIDAPI with CMake:
 ```sh
 cd <build dir>
 # configure the build
-cmake -GNinja <HIDAPI source dir>
+cmake <HIDAPI source dir>
 # build it!
+cmake --build .
+# install library; by default installs into /usr/local/
+cmake --build . --target install
+# NOTE: you need to run install command as root, to be able to install into /usr/local/
```

Comment:
This would use the source tree as the build directory.
```
cmake -S . -B build
```
creates the directory `build` if it does not exist

```
cmake --build build
cmake --install build
```

Reply:
> This would use the source tree as the build directory.

Of course it wouldn't. It will create a build tree under _current_ directory, which by `cd <build dir>` is any directory on the filesystem.

> cmake -S . -B build

This creates a build directory _inside the source tree_, which by my experience is almost as bad as using source tree for build tree.<br>
I'm using a software that scans all files under the source tree to index symbols (automatically) and if the build directory is under source tree - it scans it too, during the build as the build goes... So I prefer (and always recommend) to move build directorry somewhere outside the source tree completely.